packages/recorder/src/messageBus.ts

```diff
@@ -0,0 +1,13 @@
+import { EventEmitter } from 'events';
+
+import { VideoQueueItem } from './types';
+
+export class ManagerMessageBus extends EventEmitter {
```

Comment:
i think EventEmitter can accept generic type parameters to enforce strong type on what events are allowed

Reply:
hrm I couldn't find anything in the built-in @types stuff... link to ref? It seemed on search that a lot of people rolled their own to do something like that